server/flaskr/blueprints/meeting_agenda.py

```
@@ -45,13 +47,24 @@ def create_meeting_agenda(**obj):
     if request.method == "POST":
         previousMeetingId = data.get("previousMeetingId", -1)
         meeting = MeetingDataHandler.create_meeting_agenda(previousMeetingId=previousMeetingId, **kwargs)
+
+        if status == "planned":
+            NotificationService.add_notification(NotificationType.MeetingStart.value, g.org_id, meeting)
+
         return (jsonify(meeting.to_dict()), 201) if meeting is not None else ("", 400)
 
     elif request.method == "PUT":
```

Ruff (0.15.20)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)